VSCode/armsnippets.json

```diff
@@ -1081,4 +1081,35 @@
         "}"
     ],
     "description": "Windows Virtual Machine"
-}
+},
+"OMS Solution": {
+    "prefix": "arm-oms-solution",
+    "body": [
+        "{",
+        "   \"name\": \"[concat(${solutionName}, [${workSpaceName}])]\",",
+        "   \"location\": \"${West Europe}\",",
+        "   \"tags\": {  },",
+        "   \"type\": \"Microsoft.OperationsManagement/solutions\",",
+        "   \"apiVersion\": \"2015-11-01-preview\",",
```

Comment:
It's the latest one, see attached.

![image](https://cloud.githubusercontent.com/assets/18283616/25548234/ba93ccaa-2c6b-11e7-85a7-0dbca50154d6.png)